app/__init__.py

```diff
@@ -71,7 +71,12 @@ def create_app():
 def init_app(app):
     @app.before_request
     def required_authentication():
-        if request.path != url_for('status.show_status'):
+        no_auth_req = [
```

Comment:
Currently the callbacks are not authed. I think AWS supports this - I need to talk to firetext and other partners to see if they do to.

Note the risk is small (need correct JSON/ID's to make an update) but not ideal.